start.js

```diff
@@ -17,6 +17,21 @@ if (!envExists) {
   .pipe(fs.createWriteStream(path.join(__dirname, '/.env')))
 }
 
+// Create template session data defaults file if it doesn't exist
+const sessionDefaultsDirectory = path.join(__dirname, '/app/data')
+const sessionDefaultsFile = path.join(sessionDefaultsDirectory, '/session-defaults.js')
+const sessionDefaultsExists = fs.existsSync(sessionDefaultsFile)
+
+if (!sessionDefaultsExists) {
+  console.log('Creating template session data defaults file')
+  if (!fs.existsSync(sessionDefaultsDirectory)) {
```

Comment:
We will be adding a separate `global-data.js` file also in this directory. I wonder if we can check for the directory separately than checking for the session defaults file.

Reply:
I think we should refactor in that PR

Reply:
Ok, I can refactor this later.